subsys/bluetooth/host/classic/rfcomm.c

```diff
@@ -1545,7 +1545,7 @@ static int rfcomm_recv(struct bt_l2cap_chan *chan, struct net_buf *buf)
 	}
 
 	if (buf->len < (hdr_len + msg_len + sizeof(fcs))) {
-		LOG_ERR("Too small RFCOMM information (%d < %d)", buf->len,
+		LOG_ERR("Too small RFCOMM information (%u < %zu)", buf->len,
 			hdr_len + msg_len + sizeof(fcs));
```

Comment:
Does adding a non-size_t with a size_t always result in a size_t?

Reply:
I think the compiler will round up to the biggest type in the calculation, which in this case is `size_t`